judge/contest_format/ioi.py

```diff
@@ -0,0 +1,91 @@
+from datetime import timedelta
+from django.core.exceptions import ValidationError
+from django.db.models import Max, Min
+from django.template.defaultfilters import floatformat
+from django.urls import reverse
+from django.utils.html import format_html
+from django.utils.safestring import mark_safe
+from django.utils.translation import gettext_lazy
+
+from judge.contest_format.default import DefaultContestFormat
+from judge.contest_format.registry import register_contest_format
+from judge.utils.timedelta import nice_repr
+
+
+@register_contest_format('ioi')
+class IOIContestFormat(DefaultContestFormat):
+    name = gettext_lazy('IOI')
+    config_keys = ('no_cumtime',)
+    '''
+        no_cumtime: Specify True if time penalities are to be ignored. Defaults to False.
+    '''
+
+    @classmethod
+    def validate(cls, config):
```

Comment:
I'd have this entire function under an `if config is None: return` to reduce the overall indentation.